streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -1858,6 +1801,26 @@ object ZStream extends ZStreamPlatformSpecific {
   final def fromEffect[R, E, A](fa: ZIO[R, E, A]): ZStream[R, E, A] =
     managed(fa.toManaged_)
 
+  /**
+   * Creates a stream from an [[InputStream]].
+   */
+  final def fromInputStream[R, E, A](is: InputStream[R, E, A]): ZStream[R, E, A] =
```

Comment:
Why not implement in terms of `fromInputStreamManaged`?

Reply:
Yes, good idea!